### PR TITLE
Support no tracker tags and extra tags on race existing

### PR DESCRIPTION
**Pull request by fireph**
The flags `--no-tracker-tags` and `--extra-tags` were only supported with `qbit-race add` and not with `qbit-race race`. This PR adds this feature to `qbit-race race` with the same syntax. I didn't add support for setting the category because it doesn't really make sense to change the category after the torrent has already started.

### CODE REVIEW

**Comment by fireph** [comment]
I also can't really test and just get this error when trying to run anything:
```
node:internal/modules/cjs/loader:959
  throw err;
  ^

Error: Cannot find module '/home/user/qbit-race/bin/index.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Module._load (node:internal/modules/cjs/loader:804:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```
Is there any way to fix this issue? Would like to run locally until the npm package gets updated.

**Comment by ckcr4lyf** [comment]
Thanks for the MR. Let me test it offline once. 

Could you remove the changes made in `package-lock` ? I will update the versioning myself when I make the release.

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "qbit-race",
-  "version": "2.0.0-alpha.11",
+  "version": "2.0.0-alpha.13",
```
Comment (ckcr4lyf) [inline package-lock.json:3]:
```suggestion
  "version": "2.0.0-alpha.11",
```

File: package-lock.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "qbit-race",
-      "version": "2.0.0-alpha.11",
+      "version": "2.0.0-alpha.13",
```
Comment (ckcr4lyf) [inline package-lock.json:9]:
```suggestion
      "version": "2.0.0-alpha.11",
```

File: package-lock.json (hunk 1 above)
Comment (ckcr4lyf) [reply package-lock.json:3]: Actually, I see it is an inconsistency between the existing package.json and package-lock. Let me fix it in a separate MR so this diff goes away.

**Review by ckcr4lyf — APPROVED** [review]
Fixed externally via #46 . Thanks!

**Comment by fireph** [comment]
> Thanks for the MR. Let me test it offline once.
> 
> Could you remove the changes made in `package-lock` ? I will update the versioning myself when I make the release.

updated to remove the package-lock changes!

**Comment by ckcr4lyf** [comment]
LGTM, thanks.

**Pull request merged**